src/control/cmd/dmg/pool.go

```diff
@@ -56,12 +56,12 @@ type PoolCreateCmd struct {
 	Properties    PoolSetPropsFlag `short:"P" long:"properties" description:"Pool properties to be set"`
 	ACLFile       string           `short:"a" long:"acl-file" description:"Access Control List file path for DAOS pool"`
 	Size          string           `short:"z" long:"size" description:"Total size of DAOS pool or its percentage ratio (auto)"`
-	TierRatio     string           `short:"t" long:"tier-ratio" default:"6,94" description:"Percentage of storage tiers for pool storage (auto)"`
+	TierRatio     string           `short:"t" long:"tier-ratio" default:"6,94" description:"Percentage of SCM,NVMe storage tiers for pool storage (auto)"`
```

Comment:
As above, this wording was changed to be more generic in anticipation of adding support for tiers other than SCM and NVMe.

Reply:
i'm fine to revert this ... the goal was to be specific about what the entries in the comma-separated list are. but if this should stay generic (e.g. tier1,2,3)  then changing this text makes no sense.

Reply:
As I said in my other comment, perhaps a compromise which documents the current default would work. It would achieve the goal of describing how the flag is supposed to be used without closing the door to new use cases.

Reply:
ok, fixed